src/tokens/Assets.tsx

```diff
@@ -575,6 +575,7 @@ const styles = StyleSheet.create({
   },
   nftsContentContainer: {
     alignItems: 'flex-start',
+    paddingLeft: Spacing.Thick24,
```

Comment:
should this also be paddingHorizontal to be consistent?, although I guess it makes no difference since we also calculate the exact width of the nft image accounting for padding.